site/components/src/main/java/uk/nhs/digital/intranet/utils/AccessTokenEncoder.java

```diff
@@ -0,0 +1,40 @@
+package uk.nhs.digital.intranet.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.nhs.digital.intranet.model.AccessToken;
+
+import java.io.*;
+import java.util.Base64;
+
+public class AccessTokenEncoder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AccessTokenEncoder.class);
+
+    public String encode(final AccessToken accessToken) {
+        try {
+            final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+            final ObjectOutputStream objectOutputStream = new ObjectOutputStream(byteArrayOutputStream);
+            objectOutputStream.writeObject(accessToken);
+            objectOutputStream.close();
+            return Base64.getEncoder().encodeToString(byteArrayOutputStream.toByteArray());
+        } catch (final IOException e) {
+            LOGGER.error("Could not encode access token.", e);
+            return null;
+        }
+    }
+
+    public AccessToken decode(final String encodedString) {
```

Comment:
It's only a style suggestion, but I would have made this method static.

Reply:
Same comment as for the `encode` method